app/views/Partners/PartnersCustomUrlScreen.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState, useCallback, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { StyleSheet, View, TextInput, Alert } from 'react-native';
+import {
+  NavigationParams,
+  NavigationScreenProp,
+  NavigationState,
+} from 'react-navigation';
+
+import { NavigationBarWrapper } from '../../components/NavigationBarWrapper';
+
+import { useDispatch, useSelector } from 'react-redux';
+import getHealthcareAuthoritiesAction from '../../store/actions/healthcareAuthorities/getHealthcareAuthoritiesAction';
+import { Typography } from '../../components/Typography';
+
+import { Button } from '../../components/Button';
+import { Colors } from '../../styles';
+
+type PartnersEditScreenProps = {
+  navigation: NavigationScreenProp<NavigationState, NavigationParams>;
+};
+
+const PartnersScreen = ({
+  navigation,
+}: PartnersEditScreenProps): JSX.Element => {
+  const { t } = useTranslation();
+  const [url, setUrl] = useState('');
+  const didMountRef = useRef(false);
+
+  const dispatch = useDispatch();
+
+  // This is testing code only,we omit selector:
+  const fetchCustomStatus = useSelector(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (state: any) => state.healthcareAuthorities.request.status,
```

Comment:
bit of a code smell here? should we be maintaining an AppState type?

Reply:
we have a `RootState` type that the individual selectors use, I can add that here